.github/workflows/feature-request.yml

```diff
@@ -0,0 +1,38 @@
+name: Feature Request Manager
+on:
+  schedule:
+    - cron: 40 11 * * * # Run at 11:40 AM UTC (3:40 AM PST, 4:40 AM PDT)
+  issues:
+    types: [milestoned]
+
+jobs:
+  main:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Actions
+        if: github.event_name != 'issues' || contains(github.event.issue.labels.*.name, 'Feature Request')
+        uses: actions/checkout@v2
+        with:
+          repository: 'microsoft/vscode-github-triage-actions'
+          path: ./actions
+          ref: v35
+      - name: Install Actions
+        if: github.event_name != 'issues' || contains(github.event.issue.labels.*.name, 'Feature Request')
+        run: npm install --production --prefix ./actions
+      - name: Run Feature Request Manager
+        if: github.event_name != 'issues' || contains(github.event.issue.labels.*.name, 'Feature Request')
+        uses: ./actions/feature-request
+        with:
+          candidateMilestoneID: 30
+          candidateMilestoneName: Triage
+          backlogMilestoneID: 28
+          featureRequestLabel: Feature Request
+          upvotesRequired: 2
+          numCommentsOverride: 20
+          initComment: "This feature request is now a candidate for our backlog. The community has 60 days to upvote the issue. If it receives 20 upvotes we will move it to our backlog. If not, we will close it.\n\nHappy Coding!"
+          warnComment: "This feature request has not yet received the 2 community upvotes it takes to make it to our backlog. 10 days to go.\n\nHappy Coding!"
```

Comment:
I've never really been a fan of the "Happy Coding" signature that the VS Code team uses.  But it seems somwhat more out of place when responding to an issue with bad news such as "sorry, your feature request is not going to be addressed."

Just my opinion.